mungegithub/mungers/release-note-label.go

```diff
@@ -144,20 +144,15 @@ func (r *ReleaseNoteLabel) Munge(obj *github.MungeObject) {
 		return
 	}
 
+	releaseNoteLabelNeededAdded := false
 	if !obj.HasLabel(releaseNoteLabelNeeded) {
 		obj.AddLabel(releaseNoteLabelNeeded)
+		releaseNoteLabelNeededAdded = true
 	}
 
-	if !obj.HasLabel(lgtmLabel) {
-		return
+	if releaseNoteLabelNeededAdded && obj.HasLabel(lgtmLabel) {
+		obj.WriteComment(releaseNoteBody)
 	}
-
-	if obj.HasLabel(doNotMergeLabel) {
-		return
-	}
-
-	obj.WriteComment(releaseNoteBody)
-	obj.AddLabel(doNotMergeLabel)
```

Comment:
Then the comment will almost never be added. The first run will set the variable to false, then see there isn't the needed label, and then add it. But the PR won't have LGTM so the comment won't be written. Then the next run will have the variable set to false, the PR *does* have the needed label so the variable won't be set to true, and then even if the PR has LGTM, the variable won't be true. Still no comment.